pallets/ema-oracle/src/lib.rs

```diff
@@ -243,7 +294,10 @@ impl<T: Config> Pallet<T> {
 		assets: (AssetId, AssetId),
 		oracle_entry: OracleEntry<BlockNumberFor<T>>,
 	) -> Result<(), ()> {
-		if !T::OracleWhitelist::contains(&(src, assets.0, assets.1)) {
+		if !(T::OracleWhitelist::contains(&(src, assets.0, assets.1))
+			|| WhitelistedAssets::<T>::get().contains(&(src, (assets.0, assets.1))))
```

Comment:
i dont like the way this is tacked on, from configuration of the pallet i would expect that only `OracleWhitelist` controls it, nothing else, therefore this list should be only contains implementation (default ()) so if you override it in config, it wont be utilized

Reply:
so for our case you would basically implement extension in the runtime config, not other way around